externs/browser/w3c_serviceworker.js

```diff
@@ -570,33 +603,72 @@ function ExtendableEvent(type, opt_eventInitDict) {}
 ExtendableEvent.prototype.waitUntil = function(f) {};
 
 /**
- * @typedef {{
- *   bubbles: (boolean|undefined),
- *   cancelable: (boolean|undefined)
- * }}
+ * Defined for the forward compatibility across the derived events
+ * 
+ * @record
+ * @extends {EventInit}
+ * @see https://www.w3.org/TR/service-workers/#extendable-event-init-dictionary
  */
-var ExtendableEventInit;
+function ExtendableEventInit() {};
+
+/**
+ * TODO(vobruba-martin): Remove this property after the issue #1926 on GitHub is resolved
+ * or after a new property is added.
+ * @see https://github.com/google/closure-compiler/issues/1926
+ * @type {undefined}
+ */
+ExtendableEventInit.prototype.__do_not_use_this_dummy_property;
 
 /**
- * @see http://www.w3.org/TR/service-workers/#install-event-interface
- * @constructor
  * @param {string} type
- * @param {InstallEventInit=} opt_eventInitDict
+ * @param {!ExtendableMessageEventInit=} opt_init
+ * @constructor
  * @extends {ExtendableEvent}
+ * @template T
+ * @see https://www.w3.org/TR/service-workers/#extendablemessage-event-interface
  */
-function InstallEvent(type, opt_eventInitDict) {}
+function ExtendableMessageEvent(type, opt_init) {};
```

Comment:
`ExtendableMessageEvent` is in this file now, so this definition can be dropped.